package-lock.json

```diff
@@ -16842,6 +16845,17 @@
         "react": ">= 16.8"
       }
     },
+    "node_modules/react-easy-swipe": {
```

Comment:
yyyyy po co to?????

Reply:
Ogólnie zajebiście ważne, ale nie wiem, dlaczego - bez tej konkretnej linijki apka nie chce wstać... :/